app/utils.js

```diff
@@ -137,11 +137,29 @@ async function findTopStories() {
     }
 }
 
+async function removeOrphanedImages() {
+    const dir = getImageDirectory(ARTICLE)
+    const ls = await fs.promises.readdir(dir)
+    const files = ls.filter(file => file.includes(JPEG))
+    const fileNames = files.map(file => file.split(`.${JPEG}`)[0])
+
+    fileNames.forEach(async filename => {
+        let query = null
+        if(filename.length === 24) query = {_id: filename}
```

Comment:
We should likely keep these magical numbers at least at the top of the file as constants.